examples/rapids/mlflow_project/README.md

```diff
@@ -0,0 +1,39 @@
+### Train and Publish Locally With RAPIDS and MLFlow
```

Comment:
```suggestion
### Train and Deploy Models with RAPIDS and MLflow
```

Seems like model training & subsequent deployment with MLflow are the focus of the example? Although correct me if I'm wrong :)